concepts/list-methods/about.md

```diff
@@ -176,146 +176,27 @@ You can also provide `start` and `end` indices to search within a specific secti
 
 ## Making Copies
 
-Remember that variables in Python are labels that point to underlying objects and `lists` are _container_ objects that hold references to their collected items.
+Remember that variables in Python are labels that point to underlying objects and `lists` are _container_ objects that only hold references to their collected items.
 
-Assigning a `list` object to a new variable _name_ does not copy the object or its data. Any change made to the items in the `list` using the new variable name will also _impact the original_.
+Assigning a `list` object to a new variable _name_ does not copy the object or any of its refrenced data. Any change made to the items in the `list` using the new variable name will also _impact the original_.
 
-To avoid this complicaton, you must make a `shallow_copy` via `list.copy()` or slice. A `shallow_copy` will create a new `list` object, but **will not** create new objects for the contained list _items_. However, this type of copy will be enough for you to add or remove items from the two `list` objects independantly. (More about the differences between a shallow_copy and a deep_copy a little later).
+`<list>.copy()` will create a new `list` object, but **will not** create new objects for the referenced list _elements_. This type of copy is usually enough to add or remove items from  two `list` objects independantly and avoid problems. But if there is a chance that the elements of the `list` might be accidentially mutated and a full copy of **all** refrences and objects is needed, [`copy.deepcopy()`][deepcopy] in the `copy` module should be used.
```

Comment:
```suggestion
`<list>.copy()` will create a new `list` object, but **will not** create new objects for the referenced list _elements_. This is called a shallow copy. This type of copy is usually enough when you want to add or remove items from one of the `list` objects without modifying the other. But if there is a chance that the elements of the `list` might accidentally be mutated, a full copy of **all** references and objects should be use. [`copy.deepcopy()`][deepcopy] in the `copy` module should be used for this.
```

Reply:
oof.  Decided to do a little re-arrange.  Self-suggested/edited.